### DISCUSSION TITLE
Device request mistake

### DISCUSSION THREAD

**Discussion by OldAssComputer** [andrew-codechimp/HA-Battery-Notes#2078] [General]
I put in a request for a couple of devices a few days again and I did not input the data correctly.   I have resubmitted since, and the resubmissions now work. How can I have the bad ones pulled out of the database so they don’t clog up the list and make it longer then it has to be?

**Comment by andrew-codechimp** [comment]
If you can comment with the links to the closed issues you want removed it'll probably be easiest. I'll remove them manually.

**Reply by OldAssComputer** [reply]
https://github.com/andrew-codechimp/HA-Battery-Notes/issues/2052
And
https://github.com/andrew-codechimp/HA-Battery-Notes/issues/2054

Thank you,

I was still learning at the time.

**Comment by andrew-codechimp** [comment]
Thanks, these have now been removed.
